apps/rush/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Change Log - @microsoft/rush
 
-This log was last generated on Fri, 21 Jul 2017 22:34:44 GMT and should not be manually modified.
+This log was last generated on Wed, 30 Aug 2017 18:46:19 GMT and should not be manually modified.
+
+## 3.0.15
+Wed, 30 Aug 2017 18:24:39 GMT
+
+### Patches
+
+- Add tar as a dependency of Rush which is required work for replacing the temp_modules/*/package.json files with TGZ files
+- Add repositoryUrl to RushConfiguration to track remote repository
+- Use the new Json API from node-core-library
+- Add two new methods to ChangeFile class
```

Comment:
Rush is a tool -- we should describe these changes in terms of what the tool does.  Nobody cares about our source code refactoring.

Reply:
The change log is a combined log of both rush and rush-lib.

---
In reply to: [136168002](https://github.com/Microsoft/web-build-tools/pull/338#discussion_r136168002) [](ancestors = 136168002)